internal/stackdriver/stackdriver.go

```diff
@@ -43,6 +45,50 @@ func NewAPIClient(ctx context.Context, project string) (*API, error) {
 	}, nil
 }
 
+// RequestCount count returns the number of requests for the given offset and
+// query.
+func (a *API) RequestCount(ctx context.Context, query Query, offset time.Duration) (int64, error) {
+	logger := util.LoggerFromContext(ctx)
+	query = addFilterToQuery(query, "metric.type", requestCount)
+	endTime := time.Now()
+	startTime := endTime.Add(-1 * offset)
+	offsetString := fmt.Sprintf("%fs", offset.Seconds())
+
+	req := a.MetricClient.Projects.TimeSeries.List("projects/" + a.Project).
+		Filter(query.Query()).
+		IntervalStartTime(startTime.Format(time.RFC3339Nano)).
+		IntervalEndTime(endTime.Format(time.RFC3339Nano)).
+		AggregationAlignmentPeriod(offsetString).
+		AggregationPerSeriesAligner("ALIGN_DELTA").
+		AggregationGroupByFields("resource.labels.service_name").
+		AggregationCrossSeriesReducer("REDUCE_SUM")
+
+	logger.Debug("querying request count from Cloud Monitoring API")
+	resp, err := req.Do()
+	if err != nil {
+		return 0, errors.Wrap(err, "error when retrieving time series")
+	}
+	if len(resp.ExecutionErrors) != 0 {
+		for _, execError := range resp.ExecutionErrors {
+			loggerWith.WithField("message", execError.Message).Warn("execution error occurred")
```

Comment:
I think this is meant to be called `logger`, so the github actions’ message is correct?

Reply:
Yeah, problems with my editor after the Crostini VM has been open for some time.